### PR TITLE
pre-release vignettes review

### PR DESCRIPTION
**Pull request by kartikeyakirar**
PR to  https://github.com/insightsengineering/teal.modules.general/issues/640

### CODE REVIEW

File: vignettes/using-data-table.Rmd [hunk 1]
```diff
@@ -11,13 +11,13 @@ vignette: >
...
 various types of datasets using the data table module `tm_data_table()`:
 
-1. Load Libraries
+1. Load libraries
 2. Create data sets
 3. Create an `app` variable
-4. Run the App
+4. Run the app
 
 ## 1 - Loading libraries
```
[the remark this answers is not in the document]
Comment (kartikeyakirar) [reply vignettes/using-data-table.Rmd]: [ca77ca1](https://github.com/insightsengineering/teal.modules.general/pull/681/commits/ca77ca13362585ef3139bcbc76047206c87d8764)

**Comment by kartikeyakirar** [comment]
> I was testing if all the apps fro the vignettes worked after the modules as a variable change.
> All the example apps worked except for the example in the vignette using-scatterplot-matrix.Rmd
> 

@vedhav, I reviewed the` tm_g_scatterplotmatrix` function and noticed that it uses `teal.transform::is_single_dataset` on the variables parameter. In the case of the "Multiple wide datasets" module configuration, we currently utilize `ADSL` twice, whereas previously we used `ADSL2`. The `teal.transform::is_single_dataset` function checks the data names in the variables parameter and identifies only a single name, "ADSL," which leads to this problem. I propose removing this `mod2` to resolve the issue. as we had only single wide dataset now.

**Review by vedhav — APPROVED** [review]
Thanks @kartikeyakirar LGTM now.

**Comment by m7pr** [comment]
@kartikeyakirar does this close https://github.com/insightsengineering/teal.modules.general/issues/640 ?

**Comment by kartikeyakirar** [comment]
yes

**Pull request merged**